scanner/matcher/matcher.go

```diff
@@ -13,23 +13,27 @@ import (
 )
 
 // Matcher represents a vulnerability matcher.
-type Matcher struct {
+type Matcher interface {
```

Comment:
I know probably we are not there yet. But somewhere in the matcher we need to add a functionality to import vulns from google storage.

Reply:
we can use `ClairCore/libvuln/updates.go =>OfflineImport()`

Reply:
Yes, the Matcher will be introduced in another episode. ;-)